buildSrc/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'groovy'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    localGroovy()
+}
+
+gradlePlugin {
+    plugins {
+        simplePlugin {
+            id = 'org.terasology.gradlegoo'
+            implementationClass = 'org.terasology.gradlegoo.GradleGooPlugin'
+        }
```

Comment:
this buildSrc stuff might get streamlined by the upcoming [gradle 6.4 feature "precompiled Groovy DSL script plugins"](https://docs.gradle.org/6.4-rc-4/release-notes.html#precompiled-groovy-dsl).

or by using Kotlin. But I've been trying to be good and resisting introducing Kotlin to the codebase so I don't make more than **one** major dependency change per pull request.